src/components/contact/index.tsx

```diff
@@ -0,0 +1,35 @@
+"use client";
+
+import { FOOTER_LINKS } from "@/config/constants";
+
+import { ContactForm } from "./form";
+
+export function Contact() {
+  return (
+    <section className="bg-primary text-primary-foreground">
```

Comment:
jak dodasz to do footera to to wtedy bedzie zbedne

```suggestion
    <section>
```